dash/response.py

```diff
@@ -0,0 +1,58 @@
+"""
+Extended Flask Response for use inside callbacks
+"""
+import json
+import plotly
+from flask import Response
+from . import exceptions
+
+
+# pylint: disable=too-many-ancestors
+class DashResponse(Response):
+    """
+    Flask Response extended with option to convert a regular response to
+    valid Dash json-encoded component.
+
+    Return a `DashResponse` object from a Dash callback in order to set
+    other properties of the response like headers or cookies.
+    """
+    def __init__(self, response, **kwargs):
+        self.dash_response = response
```

Comment:
I'd like the output value to be property that does the `jsonify_response` automatically when set so we don't have to call that from dash. A benefit would be that the stacktrace would be closer to where the output_value was set and easier to debug for the user.